### PR TITLE
Add kubemark-100 for testing out new performance measurements

### PR DESCRIPTION
**Pull request by gmarek**
cc @spxtr

### CODE REVIEW

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml [hunk 1]
```diff
@@ -55,6 +55,24 @@
...
+            description: 'Run small-ish kubemark cluster to continously run performance experiments'
+            timeout: 160
+            cron-string: 'H H/2 * * *'
+            job-env: |
+                export E2E_NAME="kubemark-100"
+                export PROJECT="k8s-jenkins-kubemark"
+                export E2E_TEST="false"
+                export USE_KUBEMARK="true"
+                export KUBEMARK_TESTS="\[Feature:Performance\]"
+								export CREATE_SERVICES="true"
```
Comment (spxtr) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: Tabs aren't allowed.

Comment (gmarek) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: Done.

**Comment by gmarek** [comment]
@wojtek-t PTAL

**Comment by gmarek** [comment]
@k8s-bot unit test this issue: #IGNORE
(swagger specs)

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml [hunk 2]
```diff
@@ -55,6 +55,24 @@
                 export KUBE_GCE_ZONE="us-central1-f"
                 export KUBEMARK_MASTER_SIZE="n1-standard-1"
                 export KUBEMARK_NUM_NODES="5"
+        - '100-gce':
+            description: 'Run small-ish kubemark cluster to continously run performance experiments'
+            timeout: 160
+            cron-string: 'H H/2 * * *'
```
Comment (wojtek-t) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: Do we really need to run it every 2 hours?
How about 3 times a day (or maybe 4?).

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml [hunk 3]
```diff
@@ -55,6 +55,24 @@
                 export KUBE_GCE_ZONE="us-central1-f"
                 export KUBEMARK_MASTER_SIZE="n1-standard-1"
                 export KUBEMARK_NUM_NODES="5"
+        - '100-gce':
+            description: 'Run small-ish kubemark cluster to continously run performance experiments'
+            timeout: 160
```
Comment (wojtek-t) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: How long does it really take?

Comment (wojtek-t) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: I'm not saying it's wrong - just want to know how long it really takes.

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml (hunk 2 above)
Comment (gmarek) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: Done.

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml (hunk 3 above)
Comment (gmarek) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: Don't know - increased to 240.

**Comment by wojtek-t** [comment]
LGTM

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml [hunk 4]
```diff
@@ -55,6 +55,24 @@
                 export KUBE_GCE_ZONE="us-central1-f"
                 export KUBEMARK_MASTER_SIZE="n1-standard-1"
                 export KUBEMARK_NUM_NODES="5"
+        - '100-gce':
+            description: 'Run small-ish kubemark cluster to continously run performance experiments'
```
Comment (php-coder) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: s/continously/continuously/ ?

**Pull request merged**